dev/PushNotifications/PushNotificationBackgroundTask.cpp

```diff
@@ -23,8 +29,17 @@ namespace winrt
 void PushNotificationBackgroundTask::Run(winrt::IBackgroundTaskInstance const& taskInstance)
 {
     auto appProperties = winrt::CoreApplication::Properties();
-    winrt::PushNotificationReceivedEventArgs activatedEventArgs = winrt::make<winrt::Microsoft::Windows::PushNotifications::implementation::PushNotificationReceivedEventArgs>(taskInstance);
-    appProperties.Insert(ACTIVATED_EVENT_ARGS_KEY, activatedEventArgs);
+    if (PushNotificationHelpers::IsPackagedAppScenario())
```

Comment:
None of the code in IBackgroundInstance::Run should be modified if this scenario is working correctly. 

The only difference is how the IBackgroundInstance is composed: In one scenario, BI composes it and sends it down. In the other, it is LRP. This snippet should be agnostic of that.

Reply:
The constructor for PushNotificationReceivedEventArgs with an IBackgroundTaskInstance is only used if BI is available. Unpackaged scenarios use byte*+length | std::wstring. This is because we don't have a true IBackgroundTaskInstance and why it has to be separated.

Reply:
AS discussed, we can create a mock BackgroundTaskInstance with the mock wrappers. We don't need to change this code if we do that correctly.